app/services/csv_service.rb

```diff
@@ -0,0 +1,34 @@
+# -*- encoding : utf-8 -*-
+require 'csv'
+
+module RademadeAdmin
+  class CsvService
+
+    def initialize(model_info, items)
+      @model_info = model_info
+      @items = items
+    end
+
+    def to_csv
+      ::CSV.generate(col_sep: ',', encoding: 'utf-8') do |csv|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.